### ISSUE TITLE
Help Finding the Production Url 

### ISSUE THREAD

**Issue by YvaGithub** [susanBuck/e15-spring22#50]
Hello everyone, 
Can someone please help find the production url of the project? What are the steps to get it?
Anticipated thanks, 
Yvaral

**Comment by susanBuck** [comment]
Hi @YvaGithub - I see you removed the content from this post. Would you like me to delete the post?

**Comment by YvaGithub** [comment]
Yes please. I figured out my issue. I tried to delete it but I couldn't.
Thank you
